pkg/fedramp/flag.go

```diff
@@ -33,6 +33,13 @@ func AddFlag(flags *pflag.FlagSet) {
 		false,
 		"Uses the FedRAMP High OpenShift Cluster Manager API for creating clusters in AWS GovCloud regions",
 	)
+
+	flags.BoolVar(
+		&enabled,
+		"admin",
+		false,
+		"Uses the FedRAMP High OpenShift Cluster Manager API Endpoint for SREP Access",
+	)
```

Comment:
Please add `flags.MarkHidden("admin")` to hide this from end-users

Reply:
added, thanks